node_modules/.bin/acorn

```diff
@@ -0,0 +1 @@
+../acorn/bin/acorn
```

Comment:
- [ ] <b>REQUIRED</b>: This file and any other files from `node_modules` directory, should not be committed. that means you should have put node_modules in `.gitignore`. Kindly check out this [Article](https://www.positronx.io/should-i-commit-the-node_modules-folder-to-git/) to learn more about why you should not commit `node_modules`